jobrunner/tracing.py

```diff
@@ -48,6 +49,7 @@ def setup_default_tracing():
         add_exporter(ConsoleSpanExporter())
 
 
+@warn_assertions
```

Comment:
stupid question: do we need `warn_assertions` here, if we have it on `record_job_span`?